src/main/java/crazypants/enderio/conduit/item/filter/ItemFilter.java

```diff
@@ -88,7 +92,7 @@ private boolean itemMatched(ItemStack item) {
           matched = false;
         }        
       }
-      if(!matched && isOreDicMatch(i, item)) {
+      if(!matched && useOreDict && isOreDicMatch(i, item)) {
```

Comment:
I moved the condition from isOreDicMatch() into itemMatched - so that it follows the same style as the other controllable filters. The item==null check in isOreDicMatch was also redundant as it is already checked in it's only caller itemMatched